src/common.c

```diff
@@ -1421,6 +1421,9 @@ int ofi_bsock_async_done(const struct fi_provider *prov,
 	msg.msg_controllen = sizeof(ctrl);
 	ret = recvmsg(bsock->sock, &msg, MSG_ERRQUEUE);
 	if (ret < 0) {
+		if (OFI_SOCK_TRY_SND_RCV_AGAIN(ret))
```

Comment:
This is a direct call to recvmsg, so you want to use ```errno``` instead of ret.  This ends up being linux specific code.

Reply:
ah, yes. Thanks!